plugins/webp-uploads/hooks.php

```diff
@@ -777,3 +777,23 @@ function webp_uploads_render_generator() {
 	}
 }
 add_action( 'wp_head', 'webp_uploads_render_generator' );
+
+/**
+ * Adds a settings link to the plugin's action links.
+ *
```

Comment:
Missing `@since`

Reply:
Ah, right. Probably:

```suggestion
 *
```
```suggestion
 *
 * @since 1.0.6
 *
```